netstate/auth/process_api_key.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+package main
```

Comment:
also, make sure to put a blank line between the copyright statement and `package main`, otherwise the copyright statement will be interpreted as package documentation

Reply:
got it- thanks!